internal/cli/usercmd/client.go

```diff
@@ -476,17 +480,27 @@ func (c *EpinioClient) ServiceDetails(name string) error {
 	}
 	serviceDetails := resp.Details
 
-	msg := c.ui.Success().WithTable("", "")
+	msg := c.ui.Success().WithTable("", "", "Access Path")
 	keys := make([]string, 0, len(serviceDetails))
 	for k := range serviceDetails {
 		keys = append(keys, k)
 	}
 	sort.Strings(keys)
 	for _, k := range keys {
-		msg = msg.WithTableRow(k, serviceDetails[k])
+		path := ""
+		switch k {
+		case "Status":
+		case "Username":
```

Comment:
Slight problem, when service has a "Username" key, then the epinio username is not printed at all:

```
~/workspace/epinio (821-service-parameter-paths)*$ epinio service create myservice2 Username dimitris

🚢 Create Service
Name: myservice2
Namespace: workspace
| PARAMETER |  VALUE   |          ACCESS PATH          |
|-----------|----------|-------------------------------|
| Username  | dimitris | /services/myservice2/Username |

⚠️  Beware, the shown access paths are only available in the application's container

✔️  Service Saved.
Name: myservice2
Namespace: workspace
~/workspace/epinio (821-service-parameter-paths)*$ epinio service show myservice2

🚢 Service Details
Name: myservice2
Namespace: workspace
✔️  
|          |          | ACCESS PATH |
|----------|----------|-------------|
| Username | dimitris |             |

⚠️  Beware, the shown access paths are only available in the application's container

```

Also the table has no headers. First, I think we should call it "Epinio user" to make it clear that it's not a key from the service itself.
Also, since it's not a service key, maybe it shouldn't be in this looped object in the first place? It's service meta information, maybe it belongs to some other object/attribute.

Reply:
True, should move the information out of the general map. Requires changes to the response structure.
That will be a slightly larger change. Will do after the planning.

Reply:
Done, commit 37c434ba7258d68c130d1b7bba04eecc3b17ed3d

Note: Using CI for testing in the hope that it works better. Here I am running into pull errors for the linkerd images.
That does not bode well for local testing.